packages/ckeditor5-list/src/list/utils/listwalker.ts

```diff
@@ -191,47 +191,69 @@ export default class ListWalker {
 
 /**
  * Iterates sibling list blocks starting from the given node.
- *
- * @internal
- * @param node The model node.
- * @param direction Iteration direction.
- * @returns The object with `node` and `previous` {@link module:engine/model/element~Element blocks}.
  */
-export function* iterateSiblingListBlocks(
-	node: Node | null,
-	direction: 'forward' | 'backward' = 'forward'
-): IterableIterator<ListIteratorValue> {
-	const isForward = direction == 'forward';
-	const previousNodesByIndent: Array<ListElement> = []; // Last seen nodes of lower indented lists.
-	let previous = null;
+export class SiblingListBlocksIterator implements IterableIterator<ListIteratorValue> {
+	private node: Node | null;
+	private isForward: boolean;
+	private previousNodesByIndent: Array<ListElement> = [];
+	private previous: ListElement | null = null;
+	private previousNodeIndent: number | null = null;
```

Comment:
```suggestion
	private _node: Node | null;
	private _isForward: boolean;
	private _previousNodesByIndent: Array<ListElement> = [];
	private _previous: ListElement | null = null;
	private _previousNodeIndent: number | null = null;
```

Reply:
I forget about this every time 😮‍💨

Fixed